docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Nothing to see here right now
+
+Please refer to the [Readme](https://github.com/facebook/create-react-app/blob/master/README.md) or the [User Guide](https://github.com/facebook/create-react-app/blob/master/README.md#getting-started).
```

Comment:
This can just be the user guide.

Reply:
Sure, I'll just copy paste it in there for now. We should probably split it up at some point though.

Reply:
Ah sorry, I mis-read you. Yep, will fix the link